dealroom_firestore_connector/status_codes.py

```diff
@@ -1,4 +1,8 @@
-SUCCESS = 0
-ERROR = -1
-CREATED = 1
-UPDATED = 2
+from enum import Enum
+
+
+class Code(int, Enum):
```

Comment:
Minor comment but what about `StatusCode` ? just for a more descriptive name. Love this change btw.

Reply:
Changed in https://github.com/dealroom/data-firestore-connector/pull/31/commits/23d9a5ced496605ab65ca6e3b600e7e33a8f038c